app/actions/remote/entry/app.ts

```diff
@@ -27,6 +27,9 @@ export async function appEntry(serverUrl: string, since = 0, isUpgrade = false)
         }
     }
 
+    // Run data retention cleanup
+    await dataRetentionCleanup(serverUrl);
```

Comment:
soooo.. we try to clean data even before making sure we have the latests retention policies ?

Reply:
Currently we are blocking the entry to fetch the policies. I am removing `await`s as it's blocking the entry.

Since we are doing the cleanup as sql query, delete operation wouldn't reflect on UI if we decide to cleanup at later point of time when policies are loaded. We will always rely on the policy data that's there when the user opens the app for the last time.